packages/@aws-cdk/aws-apigateway/lib/authorizers/lambda.ts

```diff
@@ -0,0 +1,139 @@
+import iam = require('@aws-cdk/aws-iam');
+import lambda = require('@aws-cdk/aws-lambda');
+import { Construct, Duration, Lazy, Stack } from '@aws-cdk/core';
+import { CfnAuthorizer } from '../apigateway.generated';
+import { AuthorizationType, Method } from '../method';
+import { AuthorizerBase, AuthorizerConfig } from './authorizer-base';
+
+/**
+ * Properties for TokenAuthorizer
+ */
+export interface TokenAuthorizerProps {
+
+  /**
+   * An optional name for the authorizer. When provided, this will also be used for the physical id of the
+   * CloudFormation resource of type `AWS::ApiGateway::Authorizer`.
+   *
+   * @default - CDK will use the uniqueId assigned to this construct.
+   */
```

Comment:
@attribute?

Reply:
Not a CloudFormation return. It's just a human friendly name.